### DISCUSSION TITLE
Disconnect all clients from access point

### DISCUSSION THREAD

**Discussion by sabeehalam** [micropython/micropython#12574] [ESP8266]
Hi,
I am using ESP8266 as an access point hosting a web server that has some fields in a form. I wanted to know whether the client currently connected to an ESP8266 using the web server could be disconnected once they have  submitted the form on the web server?

Thank you

**Comment by sophiedegran** [comment]
If you want to disconnect all client from your AP, I think the best thing is to close the AP and relaunch it..

'''
ap = network.WLAN (network.AP_IF)
if ap.active ():
ap.active (False)
time.sleep (1) # put more if you want a reset AP on your client station..
and relaunch your AP
'''
Hope that help
